themes-default/slim/views/manage_episodeStatuses.mako

```diff
@@ -77,7 +77,7 @@ Set checked shows/episodes to <select name="newStatus" class="form-control form-
     <% series_id = str(cur_series[0]) + '-' + str(cur_series[1]) %>
     <tr id="${series_id}">
         <th><input type="checkbox" class="allCheck" data-indexer-id="${cur_series[0]}" data-series-id="${cur_series[1]}" id="allCheck-${series_id}" name="${series_id}-all" checked="checked" /></th>
-        <th colspan="2" style="width: 100%; text-align: left;"><app-link indexer-id="${cur_series[0]}" class="whitelink" href="home/displayShow?indexername=indexer-to-name&seriesid=${cur_series[1]}">${show_names[(cur_series[0], cur_series[1])]}</app-link> (${ep_counts[(cur_series[0], cur_series[1])]})
+        <th colspan="2" style="width: 100%; text-align: left;"><app-link data-indexer-to-name="${cur_series[0]}" class="whitelink" href="home/displayShow?indexername=indexer-to-name&seriesid=${cur_series[1]}">${show_names[(cur_series[0], cur_series[1])]}</app-link> (${ep_counts[(cur_series[0], cur_series[1])]})
```

Comment:
Can this be switched to using the app-link placeholder.

Reply:
I spoke too soon. Apparently it's changed in this branch, because of all the rebases.